### PR TITLE
Hetzner Quickstart docs

### PR DESCRIPTION
**Pull request by rmens**
Extend documentation with a QuickStart guide for Hetzner Cloud. They offer Owncast as app since some weeks.

### CODE REVIEW

**Comment by rmens** [comment]
@gabek Spent some time waiting for a delayed flight with finishing this up. Please have a look.

File: content/quickstart/hetzner/_index.md [hunk 1]
```diff
@@ -13,4 +13,11 @@ Using the Owncast app on Hetzner Cloud you can get Owncast installed and configu
...
 ## Create your new server
 
-[wip]
\ No newline at end of file
+Create your server as usual using the Hetzner Cloud Console. As an alternative to the operating system image, you can choose Owncast as app that you would like to have pre-installed. It is under the 'Apps' tab in the 'Images' section.
+
+<img width="1430" alt="hetzner1" src="https://github.com/rmens/owncast.github.io/assets/6742496/d3369234-fe13-4df6-99aa-b62d4a3f2c46">
+
+## Choose the right server type
+Hetzner recommends a CPX21 AMD server with 3 cores, but Owncast works well with the ARM servers from Hetzner too. They offer more cores for the same amount of money. For example the ARM server of the type CAX21 has 4 cores and costs less.
```
Comment (gabek) [inline content/quickstart/hetzner/_index.md:21]: Requirements depend on configuration. I'm hesitant to tell people "You must spend $x" if we don't know what they're trying to do. I understand Hetzner wants people to be upsold, but I'm not sure I feel comfortable endorsing it.

Comment (rmens) [reply content/quickstart/hetzner/_index.md:21]: Fair point. I'll remove this part and write something more general about CPU families available at Hetzner.

**Comment by gabek** [comment]
I'll have to remember to add Hetzner to the quickstart as well.

**Comment by rmens** [comment]
Tried to look at the Quickstart. But it's a bit too unfamiliar for me to be comfortable with editing. The documentation should be good.

**Comment by rmens** [comment]
@gabek The images don’t seem to fit the screen on the mobile view. I also noticed they link to the branch on my account. I won’t delete that until this is fixed. But I’m not sure how to.

**Comment by gabek** [comment]
Make sure any images you want to include are a part of the PR. Put them somewhere, change your image links, and make sure they render. You can follow other posts with images as examples.

**Pull request merged**